DESCRIPTION

```diff
@@ -1,12 +1,12 @@
 Type: Package
 Package: simIDM
 Title: Simulating Oncology Trials using an Illness-Death Model
-Version: 0.0.5.9026
+Version: 0.1.0
 Authors@R: c(
     person("Alexandra", "Erdmann", , "alexandra.erdmann@uni-ulm.de", role = c("aut", "cre")),
     person("Kaspar", "Rufibach", , "kaspar.rufibach@roche.com", role = "aut"),
-    person("Holger", "Loewe", , "holger.loewe@roche.com", role = "aut"),
-    person("Daniel", "Sabanes Bove", , "daniel.sabanes_bove@roche.com", role = "aut"),
+    person("Holger", "Löwe", , "holger.loewe@roche.com", role = "aut"),
```

Comment:
@holgstr should we update to your personal email address? since this one won't work anymore

Reply:
Oh yes, good catch! It‘s hbj.loewe@gmail.com. Could you add this pls? I‘m only on my phone atm.

Reply:
done :)